android/app/src/main/java/com/app/edonymyeon/data/service/fcm/AlarmService.kt

```diff
@@ -35,7 +34,30 @@ class AlarmService : FirebaseMessagingService() {
             alarmOn.value = true
         }
 
-        val intent = when (message.data["navigateTo"].toString()) {
+        val intent = getIntentByNotification(message)
+
+        val pendingIntent = PendingIntent.getActivity(
+            this,
+            0,
```

Comment:
```suggestion
            System.currentTimeMillis().toInt(),
```

pendingIntent에 들어가는 requestCode는 다른 PendingIntent와 구별하기 위해 사용되는, 마치 `고유값`과 같은 개념이에요. 
동일한 requestCode 가 같은 경우에는 마지막 PendingIntent 로 치환됩니다.
그래서 requestCode에 다른 값이 들어가도록 하여 구분을 지어보니,
리뷰에 남겨드렸던 문제가 해결되는 것 같아요. 
(이 부분도 반영해서 코비의 환경에서도 잘 동작하는지 테스트 한 번 해주세요!)

requestCode를 저는 다음과 같이 코드 수정을 제안드렸는데, 
주 요점은 **requestCode가 다른 pendingIntent가 오도록 하는 것**이니 더 나은 방법이 있다면 그 방법대로 해주셔도 됩니다!